types/index.d.ts

```diff
@@ -56,24 +51,45 @@ type Content =
   | 'space-around'
   | 'space-evenly';
 
-export interface LayoutProps {
-  /** default to 'row' */
-  direction?: Direction;
-  pad?: boolean | number;
-  wrap?: boolean;
+export interface BlockProps {
   grow?: boolean;
   inline?: boolean;
   flex?: boolean | number;
   /** default to 'stretch' */
-  align?: Align;
   alignSelf?: Align;
+}
+
+export interface FlexProps extends BlockProps {
+  /** default to 'row' */
+  direction?: 'row' | 'column';
+  pad?: boolean | number;
+  wrap?: boolean;
+  /** default to 'stretch' */
+  align?: Align;
   alignContent?: Content;
   /** default to 'flex-start' */
   justify?: Content;
 }
 
+declare class Flex<
+  As extends React.ElementType = 'div'
+> extends ComponentWithAs<As, FlexProps> {}
+
+declare class Block<
+  As extends React.ElementType = 'div'
+> extends ComponentWithAs<As, BlockProps> {}
+
+export type LayoutProps =
```

Comment:
what is this wizardry? display is not even in the proptypes

Reply:
yeah it is: https://github.com/4Catalyzer/layout/blob/483bc697065561048f3f092a908a4b5004fc5db6/src/css-modules/index.js#L9-L20

Reply:
ah there it is, I was looking in the proptypes file